test_typing_inspect.py

```diff
@@ -3,12 +3,42 @@
     is_optional_type, is_literal_type, is_typevar, is_classvar, get_origin,
     get_parameters, get_last_args, get_args, get_bound, get_constraints, get_generic_type,
     get_generic_bases, get_last_origin, typed_dict_keys,
-)
+    WITH_LITERAL, LEGACY_TYPING)
 from unittest import TestCase, main, skipIf, skipUnless
 from typing import (
-    Union, ClassVar, Callable, Optional, TypeVar, Sequence, Mapping,
+    Union, Callable, Optional, TypeVar, Sequence, Mapping,
     MutableMapping, Iterable, Generic, List, Any, Dict, Tuple, NamedTuple,
 )
+try:
+    from typing import ClassVar
+    WITH_CLASSVAR = True
+    CLASSVAR_GENERIC = [ClassVar[List[int]], ClassVar]
+    CLASSVAR_TYPEVAR = [ClassVar[int]]
+except ImportError:
+    WITH_CLASSVAR = False
+    CLASSVAR_GENERIC = []
+    CLASSVAR_TYPEVAR = []
+
+try:
+    class Foo(Callable[[int], int]):
+        pass
+    SUBCLASSABLE_UNIONS = True
+except:  # noqa E722,B001
+    SUBCLASSABLE_UNIONS = False
```

Comment:
Should this be named `SUBCLASSABLE_CALLABLES`? Currently the relation between the test and the variable name looks strange.

Reply:
Indeed I can not find any good reason why I used this name. Changing it.